api/test/audio_integration_test.py

```diff
@@ -34,6 +35,11 @@ def test_search(audio_fixture):
     search(audio_fixture)
 
 
+def test_search_category_filtering(audio_fixture):
+    search_by_category("audio", "music", audio_fixture)
+    search_by_category("audio", "pronuntiation", audio_fixture)
```

Comment:
Hmm, I'm not sure why this test _isn't_ failing because I think this should be:
```suggestion
    search_by_category("audio", "pronunciation", audio_fixture)
```

Maybe we have no pronunciation records in the sample data? :thinking: